java/show-callout/README.md

```diff
@@ -1,18 +1,31 @@
-# Show Callout
-Draw a callout on a map view and manage its behavior. A callout displays an Android View that contains text and/or other content. It has a leader that points to the location the callout refers to. 
+# Show callout
 
-![Show Callout App](show-callout.png)
+Show a callout with the latitude and longitude of user-tapped points.
+
+![Image of show callout](show-callout.png)
+
+## Use case
+
+Callouts are used to display temporary detail content on a map. You can display text and arbitrary UI controls in callouts.
 
 ## How to use the sample
-Tap on the screen to show a callout.
+
+Tap anywhere on the map. A callout showing the WGS84 coordinates for the tapped point will appear.
 
 ## How it works
-Converts screen coordinates to location coordinates to show the lat and long of the callout.  The content of the callout is created programmatically by an Android `TextView`.
+
+2. When the user taps, get the tapped location and create a map `Point` from it using `MapView.screenToLocation(tappedLocation)`.
```

Comment:
I don't know if it's worth changing `tappedLocation` to `screenPoint` to match the sample, or leave it as is to match what the screen point is.

Reply:
it's ok that variable names in the readme don't match those within the code. some users may only use the readme to get an overview of the workflow without opening up the code, so it would be more readable here to have "tappedLocation" than "screenPoint".